### DISCUSSION TITLE
How to sign up a gruop owned module?

### DISCUSSION THREAD

**Discussion by freddepettersson** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#88] [Q&A] [answered]
Hi
How to sign up /register a group module ?

**Comment by fjallemark** [comment, marked as answer]
This will be implemented in next release. Any person that is a data administrator in a group will also be able to register group owned modules for a layout.

**Reply by freddepettersson** [reply]
**Can it be like this or it it too open ?**
Any member of the actual group can register the gruop modules. In some cases the grup migth be a union and the person how register are also responsible for signed up modules.

Adminstator migth get to much on there hands !

/F

**Reply by fjallemark** [reply]
As for now, only data administrators of a group can register group modules and manage group members modules. Other group members can only manage their own modules. If you want a person to be able to register modules, make him a data administrator of your group. In this way you have full control of who can register group owned modules. Sounds OK?

**Reply by freddepettersson** [reply]
OK
